sdk/communication/AzureCommunication/Source/CommunicationIdentifierModel.swift

```diff
@@ -1,90 +0,0 @@
-// --------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-//
-// Code generated by Microsoft (R) AutoRest Code Generator.
-// Changes may cause incorrect behavior and will be lost if the code is
-// regenerated.
-// --------------------------------------------------------------------------
-
-import AzureCore
-import Foundation
-// swiftlint:disable superfluous_disable_command
-// swiftlint:disable identifier_name
-// swiftlint:disable line_length
-// swiftlint:disable cyclomatic_complexity
-
-/// Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user.
-public struct CommunicationIdentifierModel: Codable {
-    // MARK: Properties
-
```

Comment:
Where these models previously released in `AzureCommunication`? If so, we must call out the breaking change in the changelogs.

Reply:
They were, they are now generated inside of Chat's generated models. I don't believe they are intended to be exposed to users though. I will update the changelog on their removal